website/src/content/deployment-examples/azure.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Deploying to Microsoft Azure
+---
+
+This deployment example refers to [Microsoft Azure](https://azure.microsoft.com/).
+
+# Prerequisites
```

Comment:
We didn't mention this in any of the other examples (also with having a github account), as it's not really a requirement and usually it's documented in the service itself when we refer to the GitHub integration.

If we decide to mention it, we should do it for every example then. Personally I think we can omit it as it's kind of implied in a sense.

Reply:
Sure, got you, I removed that part 👍 I added it before only because it's explicitly mentioned in the [Azure's docs](https://docs.microsoft.com/en-us/azure/static-web-apps/get-started-portal?tabs=vanilla-javascript&pivots=github#prerequisites), and their infrastructure provisioning seems to be only working with GitHub